lib/ansible/module_utils/digital_ocean.py

```diff
@@ -99,3 +99,36 @@ def get_do_oauth_token(self):
             os.environ.get('OAUTH_TOKEN')
         if self.oauth_token is None:
             self.module.fail_json(msg='Unable to load api key: oauth_token')
+
+    def get_paginated_data(self, base_url=None, data_key_name=None, data_per_page=20, expected_status_code=200):
```

Comment:
Do we want to increase the default to 40? The images endpoint contains 78 objects so two requests would handle pulling all possible objects. The distro endpoint currently only contains a total of 28 objects, single request would capture everything. I ran a number of tests on the API and it looks like our average response would be around 600ms.

Reply:
+1 good addition to the module_utils thank you!

Reply:
Yes sure. I will make it 40 from 20.